documentation/tutorials/get-started.md

```diff
@@ -248,6 +248,9 @@ attributes do
   attribute :subject, :string do
     # Don't allow `nil` values
     allow_nil? false
+
+    # Allow this attribute to be public. By default, all attributes are private.
+    public? true
```

Comment:
This removes the warning:
```
** (Spark.Error.DslError) [Helpdesk.Support.Ticket]
 actions -> open -> accept:
  Cannot accept [:subject], because they are private attributes.
```